### PR TITLE
3/16 Translations Batch

### LINKED ISSUE #4505: Check in 3/16 translations batch
The translations from the 3/16 batch are in the translations sheet:
https://docs.google.com/spreadsheets/d/1x9xq-X61Gev2c470dPYe69tzI4azLTerZu-jfzFHmTE/edit?resourcekey=0-6T7y1Mv3Rr_u1rkmhGCeuQ#gid=1850787383
and ready to be checked in to their respective messages files.

### PR DESCRIPTION
**Pull request by MichaelZetune**
### Description

Checks in 3/16 batch of translations.

Although this diffs against `mzetune/messages-cleanup`, we will merge into `main` as soon as #4517 is submitted.

### Issue(s) this completes

Fixes #4505

### CODE REVIEW

File: server/conf/messages [hunk 1]
```diff
@@ -49,7 +55,7 @@ validation.errorAnnouncementSr=There are errors in the form. Please fix before c
 content.requiredFieldsAnnotation=Note: Fields marked with a * are required.
 
 # Message displayed below "Choose File" button to say that on the phone users can use their camera.
-content.mobileFileUploadHelp=On your phone? “Choose File” also allows you to use your phone camera to upload documents.
+content.mobileFileUploadHelp=On your phone? ???Choose File??? also allows you to use your phone camera to upload documents.
```
Comment (avaleske) [inline server/conf/messages:58]: Looks like the curly quotes didn't come through properly, but we should've been using regular quotes anyway. Can we change to regular `"` quotes?

File: server/conf/messages [hunk 2]
```diff
@@ -291,7 +300,7 @@ title.cannotCheckEligibility=At this time, we are unable to check your eligibili
 title.cannotCheckEligibilityTi=At this time, we are unable to check your client''s eligibility for the {0}.
 
 # Content of page when there is a system outage and eligibility can't be determined for the applicant.
-content.cannotCheckEligibility=We’re working to fix this issue quickly. Try again at a later time.
+content.cannotCheckEligibility=We???re working to fix this issue quickly. Try again at a later time.
```
Comment (avaleske) [inline server/conf/messages:303]: ditto for the curly apostrophe

File: server/conf/messages.so [hunk 3]
```diff
@@ -46,14 +50,15 @@ content.benefits=Hel faa''iidooyin
...
+content.description=CiviForm waxay kaa caawisaa helida faa'iidooyinka aad u qalmi karto {0}. Ku bilow adigoo markaa ka dooranaya codsiga hoos.
 content.description1=CiviForm wuxuu kuu ogolaanayaa inaad codsato faa''iidooyin badan hal mar adiga oo dib u isticmaalaya macluumaadka.
 content.description2=Ku bilow doorashada codsiga hoos.
 content.doesNotQualify=Uma qalmo
 content.eligibilityCriteria=Heerka qiimaynta u qalmida fadlan tixraac {0}
 content.foundSimilarAddress=Waanu xaqiijin kari waynay cinwaanka aad bixisay, laakiin waxaanu helnay shay la mid ah.
 content.loginDisabledPrompt=Gelitaanka koontada hadda waa la xidhay
 content.loginPrompt=Fadlan ku gal akoonkaaga Magaalada {0}
-content.mobileFileUploadHelp=Telefoonkaaga? “Choose File” sidoo kale waxuu kuu oggolaanayaa inaad isticmaasho kamarada telefoonkaaga si aad u soo geliso dukumintiyo.
+content.mobileFileUploadHelp=Telefoonkaaga? ???Choose File??? sidoo kale waxuu kuu oggolaanayaa inaad isticmaasho kamarada telefoonkaaga si aad u soo geliso dukumintiyo.
```
Comment (avaleske) [inline server/conf/messages.so:61]: Curly quotes again

**Review by avaleske — APPROVED** [review]
Just some issues with curly quotes but otherwise lgtm

File: server/conf/messages (hunk 2 above)
Comment (MichaelZetune) [reply server/conf/messages:303]: Fixed.

File: server/conf/messages (hunk 1 above)
Comment (avaleske) [reply server/conf/messages]: Instead of reverting the diff, could you change them to straight quotes, since you're in here anyway?

Comment (MichaelZetune) [reply server/conf/messages]: Done.

**Pull request merged**